Core tutorials seo

BillWagner opened the pull request:
contributes to #9440 

Most of the work was in the description and titles, with a bit in the H1 length validation.

BillWagner committed:
convert HTTP to HTTPS in include file

update links, and ms.date format

P0 cats issues.

BillWagner committed:
finish title length SEO issues.

BillWagner committed:
update descriptions for seo

BillWagner committed:
improve H1 for SEO

thse had H1s that were too short and didn't describe the intent

BillWagner committed:
fix mscustom metadata for multiple values.

JRAlexander commented on docs/core/tutorials/netcore-hosting.md:
```diff
@@ -1,11 +1,12 @@
-title: Hosting .NET Core
-description: Hosting the .NET Core runtime from native code
+title: Write a custom .NET Core runtime host
+description: Can can host the .NET Core runtime from native code to support advanced scenarios that require controlling how the .NET Core runtime works.
```
Remove a can. :)

JRAlexander reviewed (approved):
LGTM. One small item and you are good to go.

BillWagner committed:
feedback

- typo in one description
- fix tab name
- fix ms.custom tag. It's not YAML, its a comma delimited string.
